sysml.library/Systems Library/Actions.sysml

```diff
@@ -117,7 +119,10 @@ package Actions {
 	 * A SendAction is an Action used to type a SendActionUsage. It initiates an outgoingTransferFromSelf 
 	 * to a designated receiver Occurrence with a given payload.
 	 */
-	action def SendAction(in payload[0..*], in receiver: Occurrence[1]) {
+	action def SendAction {
+		in payload[0..*];
+		in receiver: Occurrence[1];
```

Comment:
Just for my understanding: what is the rationale for moving away from programming language style, positional formal parameter declarations to plain feature declarations? Is it to support alternative constructor signatures with positional parameters in the future?

Reply:
Since parameters are now just directed features on behaviors, I think it is, in general, clearer to use the directed feature notation consistently. Further, the syntax for the programming-style parameters is more limited. In this particular case, it was necessary to use the directed feature notation in order to put a default on the `receiver` parameter of `AcceptAction`. It therefore seemed more parallel to also use that notation on `SendAction`.